teuthology/suite/placeholder.py

```diff
@@ -70,7 +70,7 @@ def _substitute(input_dict, values_dict):
                     'debug osd': 25
                 }
             },
-            'log-whitelist': ['slow request'],
+            'log-whitelist': ['\(SLOW_OPS\)'],
```

Comment:
we'll need to leave the original variant here too, since teuthology master is used for all ceph branches

Reply:
good point, I'll add it back in.